hw/src/scss/_weatherWidget.scss

```diff
@@ -0,0 +1,89 @@
+@import "_variablesForWW";
+@font-face {
+    font-family: $FONTF;
+    src: $FONTFURL format("opentype");
+}
+
+@mixin bcAndTs ($color, $tsColor, $tsSize: $SHODOWSIZE, $transparency:$SHODOWTRANSPARENCY) {
```

Comment:
I would suggest renaming the mixin to something more easily understandable, i.e. "backgrounds-and-transparency". Also, the parameters in the brackets shall be put on separate lines in order to improve code readability. A word of advice:  installing "prettier" (https://prettier.io/docs/en/index.html) in your project and creating a configuration file for it(https://prettier.io/docs/en/configuration.html) will help you easily maintain code style and readability.